docs/modules/publisher-dmmigration.md

```diff
@@ -0,0 +1,143 @@
+      "QueueSize": "integer"
+    }
+  ]
+}
+```
+
+
+
+Method call's request attributes are as follows:
+
+| Attribute                        | Mandatory | Type    | Default                   | Description                                                                                  |
+|----------------------------------|-----------|---------|---------------------------|----------------------------------------------------------------------------------------------|
+| `EndpointUrl`                    | yes       | String  | N/A                       | The OPC UA Server’s endpoint URL                                                             |
+| `UseSecurity`                    | no        | Boolean | false                     | Desired opc session security mode                                                            |
+| `OpcAuthenticationMode`          | no        | Enum    | Anonymous                 | Enum to specify the session authentication.<br>Options: Anonymous, UserName                  |
+| `UserName`                       | no        | String  | null                      | The username for the session authentication<br>Mandatory if OpcAuthentication mode is UserName|
+| `Password`                       | no        | String  | null                      | The password for the session authentication<br>Mandatory if OpcAuthentication mode is UserName|
+| `DataSetWriterGroup`             | no        | String  | EndpointUrl_#             | The writer group collecting datasets defined for a certain <br>endpoint uniquely identified by the above attributes. <br>This shall be used to identify the session opened into the <br>server. The default value consists of the EndpointUrl string, <br>followed by a deterministic hash composed of the <br>EndpointUrl, security and authentication attributes.|
+| `DataSetWriterId`                | no        | String  | DataSetPublishingInterval | The unique identifier for a data set writer used to collect <br>opc nodes to be semantically grouped and published with <br>a same publishing interval. <br>When not specified a string representing the common <br>publishing interval of the nodes in the data set collection. <br>This the DataSetWriterId shall uniquely identify a data set <br>within a DataSetGroup. The unicity shall be determined <br>using the provided DataSetWriterId and the publishing <br>interval of the grouped OpcNodes.  An individual <br>subscription shall be created for each DataSetWriterId|
+| `DataSetPublishingInterval`      | no        | Integer | false                     | The publishing interval used for a grouped set of nodes <br>under a certain DataSetWriter. When defined it shall <br>override the OpcPublishingInterval value in the OpcNodes <br>if grouped underneath a DataSetWriter. |
+| `Tag`                            | no        | String  | empty                     | TODO                                                                                         |
```

Comment:
TODO

Reply:
we will add this TODO later once it is implemented.